components/engine_traits/src/cf_defs.rs

```diff
@@ -6,7 +6,7 @@ pub const CF_LOCK: CfName = "lock";
 pub const CF_WRITE: CfName = "write";
 pub const CF_RAFT: CfName = "raft";
 // Cfs that should be very large generally.
-pub const LARGE_CFS: &[CfName] = &[CF_DEFAULT, CF_WRITE];
+pub const LARGE_CFS: &[CfName] = &[CF_DEFAULT, CF_LOCK, CF_WRITE];
```

Comment:
just curious, how does large CFs being handle differently?

Reply:
The estimated size of `LARGE_CFS` are used to determine whether to split a region.

Reply:
I guess in the huge transaction scenario the prewrite stage will write many data into lock cf, and this should trigger region split.